docs/DeveloperGuide.md

```diff
@@ -558,9 +593,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 3a. User requests to open up Help Start window.
 
   * 3a1. OneShelf does **not** open a new Help Start Window.
-  
+
   Use case ends.
-    
+   
 **Use case: UC05 - Undoing a command**
```

Comment:
Can I trouble you to change line 625 of undoes a command from <u> to </u> 
If not everything is underlined until the end of the file. Thanks!

Reply:
is it you want to remove the underline? currently does not seem to have everything underlined

Reply:
ah its only in my local, ignore this